<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo_Dialog</class>
 <widget class="QDialog" name="Demo_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color widget demo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="Color_Wheel" name="color_wheel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Wheel Style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shape</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color Space</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="combo_shape"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_angle"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="combo_color"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="Color_Preview" name="color_preview"/>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Color_List_Widget" name="color_list"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Color_Preview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>Color_Wheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>Color_List_Widget</class>
   <extends>QWidget</extends>
   <header>color_list_widget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>color_wheel</sender>
   <signal>colorChanged(QColor)</signal>
   <receiver>color_preview</receiver>
   <slot>setColor(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
